invenio_rdm_records/services/services.py

```diff
@@ -22,12 +22,20 @@ class RDMRecordService(RecordDraftService):
     default_config = config.RDMRecordServiceConfig
 
 
-class RDMUserRecordsService(RecordService):
+class RDMUserRecordsService(RecordDraftService):
     """Bibliographic user records service."""
 
     config_name = "RDM_RECORDS_BIBLIOGRAPHIC_USER_RECORDS_SERVICE_CONFIG"
     default_config = config.RDMUserRecordsServiceConfig
 
+    def user_records_search(self, identity, params=None, links_config=None,
```

Comment:
I moved the newly added method here as I thought that this feature is specific to RDM needs....Also all the systemfileds are defined in this module... But, I have some doubts if we want this as part of the invenio-drafts-resources.... In that case, I just need to move the functionality in that module and provide it as core feature...